Apps/Playground/Android/app/CMakeLists.txt

```diff
@@ -63,7 +63,8 @@ target_link_libraries(BabylonNativeJNI
         Console
         Window
         ScriptLoader
-        XMLHttpRequest)
+        XMLHttpRequest
+        DOM)
```

Comment:
I'm not sure about calling this polyfill `DOM` though I'm not sure I have a better name. I don't think we will ever implement all of the Web APIs that start with DOM.